src/org/zaproxy/zap/extension/alert/ExtensionAlert.java

```diff
@@ -376,6 +380,23 @@ private void writeAlertToDB(Alert alert, HistoryReference ref) throws HttpMalfor
 
     }
 
+    private TreePath findAlertInTree(Alert alert) {

```

Comment:
Hmmmm in the two saved sessions I tested with this always returned zero. I unfortunately need to head to work and can't do anything further with this until tonight.

I can probably just use `findLeafNodeForAlert(root, alert)` which is already in AlertTreeModel.